_posts/2020-04-29-release-R119/index.md

```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: Snowplow R119 Tycho Magnetic Anomaly Two production release with new bad row format 
+title-short: Snowplow R119 Tycho Magnetic Anomaly Two
+tags: [snowplow, enrichment, bad-rows, release]
+author: Anton
+category: Releases
+permalink: /blog/2020/04/29/snowplow-release-r119/
+---
+
+We are thrilled to release [Snowplow R119 Tycho Magnetic Anomaly Two][snowplow-release].
+
+This release is about as big as they come, as it is an important milestone for the Snowplow pipeline. It marks the production ready release of a new bad rows (failed events) format and becomes the **last umbrella release** in the history of this project.
```

Comment:
Are we missing the changes we've made to code quality and testing here?

Reply:
will add a note